empress/recon_vis/recon_viewer.py

```diff
@@ -96,83 +222,358 @@ def render_parasite_helper(fig, node, recon, host_lookup, show_internal_labels,
+        host_node.layout.lower_v_track += (host_node.layout.x - node_xy[0]) / offset
+    else:
+        stop_row = host_node.left_node.layout.row
+        connect_child_to_parent(node, left_node, host_lookup, recon, fig, stop_row=stop_row)
+
+    #Draw Right node
+    if host_node.right_node.name == right_host_node.name:
+        render_curved_line_to(node_xy, right_xy, fig)
+        host_node.layout.upper_v_track += (host_node.layout.x - node_xy[0]) / offset
+    else:
+        stop_row = host_node.right_node.layout.row
+        connect_child_to_parent(node, right_node, host_lookup, recon, fig, stop_row=stop_row)
+
+
+def get_children(node, recon, parasite_lookup):
+    """
+    Gets the children of a node in the order they appear in the mapping node.
+    :param node: Node object
+    :param recon: Reconciliation Object
+    :param parasite_lookup: Dictionary with parasite node names as the key and parasite node objects as the values
+    :return A tuple consisting of the left node and right node
+    """
+    mapping_node = recon.mapping_of(node.name)
+    event = recon.event_of(mapping_node)
+    left_mapping_node = event.left
+    right_mapping_node = event.right
+    left_node_name = left_mapping_node.parasite
+    right_node_name = right_mapping_node.parasite
+
+    left_node = parasite_lookup[left_node_name]
+    right_node = parasite_lookup[right_node_name]
+
+    return left_node, right_node
+
+def render_curved_line_to(node_xy, other_xy, fig):
+    """
+    Renders a curved line from one point to another
+    :param node_xy: x and y position of a node
+    :param other_xy: x and y position of another node
+    :param fig: Figure object that visualizes trees using MatplotLib
+    """
+    mid_xy = (node_xy[0], other_xy[1])
+    fig.line(node_xy, mid_xy, PARASITE_EDGE_COLOR)
+    fig.line(mid_xy, other_xy, PARASITE_EDGE_COLOR)
+
+def render_transfer_branch(node_xy, right_xy, fig, node, host_lookup, recon, right_node):
+    """
+    Renders a transfer branch
+    :param node_xy: x and y position of a node
+    :param right_xy: x and y position of the right child of a node
+    :param fig: Figure object that visualizes trees using MatplotLib
+    :param node: Node object
+    :param host_lookup: Dictionary with host node names as the key and host node objects as the values
+    :param recon: Reconciliation object
+    :param right_node: The right node object of node
+    """
+    mapping_node = recon.mapping_of(node.name)
+    host_node = host_lookup[mapping_node.host]
+
+    child_mapping_node = recon.mapping_of(right_node.name)
+    child_host_node = host_lookup[child_mapping_node.host]
+
+    #check temporal consistency of transfer event
+    if child_host_node.parent_node.layout.col < node.layout.col:
+        #Draw right node, which is transfered
+        mid_xy = (node_xy[0], right_xy[1])          #xy coords of midpoint
+        y_midpoint = abs(mid_xy[1]+ node_xy[1])/2   #value of midpoint between mid_xy and parent node
```

Comment:
Space after `#`
Add space between `/`
Add space before and after `+`, or remove the space after `+`.
```python
y_midpoint = abs(mid_xy[1] + node_xy[1]) / 2  # value of midpoint between mid_xy and parent node
```
OR
```python
y_midpoint = abs(mid_xy[1]+node_xy[1]) / 2  # value of midpoint between mid_xy and parent node
```

Reply:
Done

Reply:
fixed